src/api.js

```diff
@@ -254,6 +254,10 @@
     connect.core.getUpstream().onUpstream(connect.AgentEvents.MUTE_TOGGLE, f);
   };
 
+  Agent.prototype.onLocalMediaStreamCreated = function (f) {
```

Comment:
add documentation.md entry for this api.

Reply:
Asked Mikbail to add entry today.